### PR TITLE
Cloudtrail, Security Hub mappings, samples v1.1.0 updates

### PR DESCRIPTION
**Pull request by floydtree**
1. Updating Cloudtrail mappings, samples to reflect v1.1.0 
2. Adding basic selector references for cloudtrail, sechub
3. Updating SecHub mappings, samples to reflect v1.1.0
4. Minor update to vpc, r53, eks mappings.

### CODE REVIEW

File: mappings/markdown/AWS/v1.1.0/Route53/README.md
Comment (adplotzk) [inline mappings/markdown/AWS/v1.1.0/Route53/README.md:1]: Change to observables looks good her

File: mappings/markdown/AWS/v1.1.0/CloudTrail/Authentication/samples/authentication.raw
Comment (adplotzk) [inline mappings/markdown/AWS/v1.1.0/CloudTrail/Authentication/samples/authentication.raw:1]: cloudtrail looks good as well

**Review by adplotzk — APPROVED** [review]
Changes look good

**Review by nathanbvail — COMMENTED** [review]
LGTM

**Pull request merged**